src/app/app-routing.module.ts

```diff
@@ -1,7 +1,11 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { CoursesListComponent } from './features/courses/courses-list/courses-list.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', redirectTo: 'courses', pathMatch: 'full' },
```

Comment:
If you'd like to redirect to courses page even if you have wrong link (http://localhost:4200/coursesadawdawdawd) add this property
{ path: '**', pathMatch: 'full', redirectTo: RootRoutes.courses }